Sources/NIO/BaseSocket.swift

```diff
@@ -203,24 +203,31 @@ extension sockaddr_storage {
     }
 }
 
+private func SetNonBlocking(socket: CInt) throws {
```

Comment:
I don't love having this as a top-level function: it'd be really good to find an appropriate type to hang it off.

Reply:
What about placing this in `System`?

Reply:
I'd be fine with that.